### DISCUSSION TITLE
Can we use quarkus redis annotations while integrating with Redisson ?

### DISCUSSION THREAD

**Discussion by nitinty** [redisson/redisson#5942] [Q&A]
Can we use quarkus redis annotations (@CacheResult, @CacheInvalidate, @CacheInvalidateAll, @Cachekey) while integrating with redisson-quarkus ? Please assist here.

**Comment by nitinty** [comment]
@mrniko please assist here

**Reply by mrniko** [reply]
It's not supported for now. Can you create a feature request regarding this task?

**Reply by nitinty** [reply]
@mrniko 
Raised the below request for the same, Please check 

https://github.com/redisson/redisson/issues/5943
